unstructured/cleaners/extract.py

```diff
@@ -63,3 +63,34 @@ def extract_us_phone_number(text: str):
     start, end = regex_match.span()
     phone_number = text[start:end]
     return phone_number.strip()
+
+
+def extract_ordered_bullets(text) -> tuple:
+    """Extracts the start of bulleted text sections up to three “sub-section”
```

Comment:
Could you mention explicitly that the tuple represents `(section, sub_section, sub_sub_section)`